cylc/flow/scheduler.py

```diff
@@ -1011,9 +1032,9 @@ def _set_stop(self, stop_mode: Optional[StopMode] = None) -> None:
         self.stop_mode = stop_mode
         self.update_data_store()
 
-    def command_release(self, task_globs: Iterable[str]) -> int:
+    def command_release(self, tasks: Iterable[str]) -> int:
```

Comment:
I think I had to change these args along with the change to the the resolvers module, to remove the unnecessary function wrappers. But I'll recheck that ...

Reply:
Right.

Reply:
I think `task_globs` still applies

Reply:
What's happened here is that Hillary has removed an intermediate function that was converting the argument names from the GraphQL schema to the argument names used in these functions.

As a result the argument must now be given the same name as used in the GraphQL schema, which is `task` rather than the more informative `task_glob`.